### PR TITLE
feat(unlock-protocol-com): Redesign the blog page and refactor internals

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

Adds the redesigned blog.

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/next.config.js [hunk 1]
```diff
@@ -59,19 +59,4 @@ module.exports = {
     })
     return config
   },
-  exportPathMap: async (defaultPathMap, { dir }) => {
```
Comment (julien51) [inline unlock-protocol-com/next.config.js:62]: Don't we need that to make sure the static pages are built ahead of time?

**Review by julien51 — COMMENTED** [review]
I really want to make sure we're not breaking static builds

File: unlock-protocol-com/next.config.js (hunk 1 above)
Comment (searchableguy) [reply unlock-protocol-com/next.config.js:62]: Nope. Next js generates a static page automatically based on the lage and posts where we need external data at build time, you can use getStaticProps function which I have done. 

https://nextjs.org/docs/advanced-features/automatic-static-optimization

https://nextjs.org/docs/basic-features/data-fetching/get-static-props

**Comment by julien51** [comment]
I checked and looks like it works as expected! LGTM! 🚀

**Review by julien51 — APPROVED** [review]

**Pull request merged**